package.json

```diff
@@ -89,8 +91,10 @@
     "lru-cache": "^7.14.0",
     "methods": "^1.1.2",
     "module-details-from-path": "^1.0.3",
+    "msgpack-lite": "^0.1.26",
```

Comment:
We shouldn't be using this outside of devDependencies as MessagePack is encoded manually.

Reply:
@piochelepiotr added this, I suspect he didn't know we already had a message pack implementation. I don't know if there are any shortcomings to our implementation that needed to be solved with a third party package. Piotr is out this week so we'll try to investigate.

Reply:
we've decided to keep this package since writing our custom message pack implementation for dsm would take too much effort at this point

Reply:
The work has already been done in https://github.com/DataDog/dd-trace-js/blob/master/packages/dd-trace/src/encode/span-stats.js